python-da/data-prep/src/app.py

```diff
@@ -4,6 +4,7 @@
 import pandas as pd
 from azure.core.exceptions import AzureError
 from azure.storage.blob import BlobServiceClient
+from .data_prep import DataPrep
```

Comment:
is . data_prep correct syntax

Reply:
Will resolve this after our API call thanks